amarillo/services/secrets.py

```diff
@@ -1,10 +1,13 @@
 from typing import Dict
 from pydantic import Field
 from pydantic_settings import BaseSettings
-
+from typing import Optional
 # Example: secrets = { "mfdz": "some secret" }
 class Secrets(BaseSettings):
     ride2go_token: str = Field(None, env = 'RIDE2GO_TOKEN')
+    # TODO: define these as required if metrics plugin is installed
+    metrics_user: Optional[str] = Field(None, env = 'METRICS_USER')
+    metrics_password: Optional[str] = Field(None, env = 'METRICS_PASSWORD')
```

Comment:
It feels a bit strange to have plugin configuration in the core settings. Shouldn't this be done by the plugins requiring additional config?

Reply:
I agree, I'll try and find a way to have this configuration on the plugin side